tests/test_project/tests/test_password_history.py

```diff
@@ -100,3 +100,257 @@ def test_change_number_hasher_iterations(self):
                     user_config__iterations=HistoryVeryStrongHasher.iterations).count(),
                 1,
             )
+
+    @override_settings(AUTH_PASSWORD_VALIDATORS=[{
+        'NAME': 'django_password_validators.password_history.password_validation.UniquePasswordsValidator',
+        'OPTIONS': {
+            'lookup_range': 3
+        }
+    }])
+    def test_lookup_range(self):
+        """
+        Tests considering the last 3 passwords and setting a valid password.
+        """
+        self.create_user(1)
+        self.user_change_password_number_of_times(user_number=1, num_of_times=4)
+        for i in range(4):
+            self.assert_password_validation_False(
+                user_number=1, password_number=i
+            )
+        self.assert_password_validation_True(user_number=1, password_number=4)
+
+    def test_lookup_range_string(self):
```

Comment:
@spokerman12, you should consider combining all these 'invalid values' test cases into a single test case where you iterate over the invalid values and do the assertions in a loop.